cypress/e2e/shared/explorerVisualizations.test.ts

```diff
@@ -77,13 +77,10 @@ describe('visualizations', () => {
 
           cy.log('can revert back to query builder mode (with confirmation)')
           cy.getByTestID('switch-to-script-editor').should('be.visible').click()
-          cy.getByTestID('flux--aggregate.rate--inject').click()
+          cy.getByTestID('flux--bottom--inject').click()
```

Comment:
I chose to inject his function because since we are in the shared folder and this test will run against both oss and cloud, we want a flux function that has the same test id for both dynamic flux and static flux.